jsconfig-lint.json

```diff
@@ -1,4 +1,9 @@
 {
   "extends": "./jsconfig.json",
-  "include": ["src/**/*", "hooks/src/**/*"]
-}
+  "include": [
+    "src/**/*",
+    "hooks/src/**/*",
+    "compat/**/*.d.ts",
+    "jsx-runtime/**/*.d.ts"
```

Comment:
compat & jsx-runtime source & test files throw up some errors yet but I think it's worth linting their types at least as these are user-exposed. Will help guard against typos and what not